File: practice/p1/prime5to50.py
output_template = " is a prime number"

def is_prime(x):
  if x < 2:
    return False
  else:
    for i in range(2, x):
      if x % i == 0:
        return False
    return True


for i in range(5,51):
  if is_prime(i):
    print(str(i) + output_template)

File: practice/p1/prime5to50.js
function is_prime(x) {
    if (x < 2)
        return false;
    else {
        var i_1;
        for (i_1 = 2; i_1 < x; i_1++) {
            if (x % i_1 === 0)
                return false;
        }
        return true;
    }
}
var i;
for (i = 5; i <= 50; i++) {
    if (is_prime(i) === true) {
        document.write(i + " is a prime number <br>");
        console.log(i + " is a prime number \n");
    }
}
